Entities/Arm.cpp

```diff
@@ -446,7 +446,8 @@ void Arm::UpdateArmFrame() {
 void Arm::Draw(BITMAP *pTargetBitmap, const Vector &targetPos, DrawMode mode, bool onlyPhysical) const {
     Attachable::Draw(pTargetBitmap, targetPos, mode, onlyPhysical);
 
-    if (!onlyPhysical && (mode == g_DrawColor || mode == g_DrawWhite || mode == g_DrawTrans) && (!m_Parent || m_pHeldMO || (!m_pHeldMO && !m_DidReach))) {
```

Comment:
I hope you've tested these changes a lot and with different stuff (i.e. all possible combination of guns, shields, tds, empty hands, etc) to be sure there're no issues. Hand draw order weirdness and ordering of these functions in arm update matters a lot as I recall.
Specifically, I don't remember what it was but there must have been some reason to not draw if the arm DidReach - the hand must have been drawing over something or other?

Reply:
The property naming and structure is messy, but I assure you I thoroughly tested it out to be sure that it works as intended.

Methinks a lot of this stuff is Dan trying to work around hands improperly appearing due to how different HeldDevices are treated, and it's all unraveling now that I'm changing the logic lol

Reply:
Good enough for me :)

And yeah, I don't disagree with you, it was super complicated to deal with edge cases. If this simplification works then I'm all for it!
I'll still give it a playtest before I merge, also to see how it feels.